### DISCUSSION TITLE
PSA: Status checks now required to pass before merging

### DISCUSSION THREAD

**Discussion by jingtang10** [google/android-fhir#246] [General]
Starting today, you will not be able to merge to master without status checks passing.

If you believe the tests are flakey and your PR is being blocked please raise the issue to the reviewers and code owners.

**Comment by pld** [comment]
Cool, reading some of the PR comments, I wonder if we could tighten or add style checks to move some of that work from manual to the CI?

**Reply by jingtang10** [reply]
currently spotless is a mandated step in github actions: https://github.com/google/android-fhir/blob/master/.github/workflows/ci.yml#L77 but it's not as strict as we'd liked. perhaps we need to investigate a little more how to make the rules stricter: https://github.com/diffplug/spotless/tree/main/plugin-gradle#kotlin

**Reply by jingtang10** [reply]
@pld i think it might be a good idea to switch to use ktfmt https://github.com/facebookincubator/ktfmt. It appears to be much stricter.
